UI/src/view/ProhibitionFormElements.js

```diff
@@ -163,12 +177,24 @@
             '</div>';
         }
 
+
+        function additionalInfomationField(prohibition) {
+
+          return '' +
+              '<label>Tarkenne:</label>' +
```

Comment:
Is it possible to have Tarkenne label and text field to be indented to the same level as Rajoitus ei koske... and dropdown menu before it?